From 7546dc06f0a08270674e25d671dd09cd322aa407 Mon Sep 17 00:00:00 2001
From: David L Kreitzer <david.l.kreitzer@intel.com>
Date: Tue, 15 Oct 2013 13:52:20 -0700
Subject: Dalvik: Enable the 64-bit integer to XMM optimization for PCG

BZ: 145401

This patch simply enables the 64-bit integer to XMM optimization.  All the
enabling functionality is already there.

Category: device-enablement
Domain: AOSP-Dalvik-Compiler-CG
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: I05e1291e53821777abf672e378fad13ee7b071bc
Orig-MCG-Change-Id: I9f9ab1c4e22c35ce4791ef7ce175b84268b266f7
Signed-off-by: Kreitzer, David L <david.l.kreitzer@intel.com>
Reviewed-on: http://android.intel.com:8080/138710
Reviewed-by: Banerji, Udayan <udayan.banerji@intel.com>
Reviewed-by: Hartley, Timothy D <timothy.d.hartley@intel.com>
Reviewed-by: lab_aqa <lab_aqa@intel.com>
Reviewed-by: Popov, Ivan G <ivan.g.popov@intel.com>
Reviewed-by: Chen, Dong-Yuan <dong-yuan.chen@intel.com>
Tested-by: Chen, Dong-Yuan <dong-yuan.chen@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/compiler/codegen/x86/pcg/PcgInterface.cpp |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/vm/compiler/codegen/x86/pcg/PcgInterface.cpp b/vm/compiler/codegen/x86/pcg/PcgInterface.cpp
index 83cfef4..d47a025 100644
--- a/vm/compiler/codegen/x86/pcg/PcgInterface.cpp
+++ b/vm/compiler/codegen/x86/pcg/PcgInterface.cpp
@@ -80,7 +80,6 @@ static void pcgConfigureTrace (CompilationUnitPCG *cUnit)
                         "eliminate_frame_pointer", "off",
                         "stack_alignment_compatibility", "0",
                         "expand_32b_idiv_irem", "false",
-                        "use_def_in_postpass_64_to_sse", "off",
                         (void*)0);
 
     if (cUnit->checkDebugMask (DebugMaskAsm) == true)
-- 
1.7.4.1

